CONTRIBUTING.md

```diff
@@ -581,18 +585,67 @@ mod tests {
 }
 ```
 
-For more detailed guidance on how to add Rust testing you can refer to the Rust
+For more detailed guidance on how to write Rust tests, you can refer to the Rust
 documentation's [guide on writing tests](https://doc.rust-lang.org/book/ch11-01-writing-tests.html).
 
-To run the Rust-space tests, do
+Rust tests are run separately from the Python tests. The easiest way to run
+them is via ``tox``, which creates an isolated venv and pre-installs ``qiskit``
+prior to running ``cargo test``:
+
+```bash
+tox -erust
+```
+
+You can also execute them directly in your own virtual environment with these
+commands (which is what the ``tox`` env is doing under the hood):
```

Comment:
There is a path to running without a virtual environment, but that's probably not good to encourage because it means people will be installing a dev version of qiskit in their system site packages. It might be good to add a sentence here explaining you should make a venv for testing in this case.

Reply:
Good point. I've reworded this section to guide users to first create and activate a virtual environment if they're running without tox.

234b6a56e847914cfb75c6462489bb1900bb4ea1